### ISSUE TITLE
Все блоки вынести в свои папки

### ISSUE THREAD

**Issue by chmnkh** [NikitaLem/first_task_cv#14]
main-content - не исключение :)

**Comment by chmnkh** [comment]
1. Инклуд миксина main блока надо в самый вверх файла вынести, это же импорт фактически
2. Чтобы index.pug стал чище, весь массив данных, который ты в main передаешь, лучше из pug файла вынести, и потом в самом файле импортить. Можешь в json вынести, например.

**Comment by chmnkh** [comment]
Ну в json ты вынес, а дальше что?) Оно же у тебя не используется совсем.
И вынес странно, почему у тебя все значения - строки?

**Comment by NikitaLem** [comment]
Да. Я это ещё не закончил, поэтому не коммитил эту ишью. На самом деле я всё ещё пытаюсь понять как импортировать json в pug файл)

**Comment by chmnkh** [comment]
Окей, в будущем тогда если коммита нет - жду)

**Comment by NikitaLem** [comment]
@chmnkh не могу понять как подключить файл с переменными к pug файлу. Подключал через include ./путь_к_файлу в самом pug файле. Пробовал через import variables from 'путь_к_файлу' в главном js файле проекта. Оба варианта не работают. Прошу, дай подсказку. :)

**Comment by chmnkh** [comment]
json можно через require прямо в переменную подключить в index.pug'e :) 
объявить данные в pug как сейчас не очень красиво, согласись, не знаю, сделал ты это от безысходности или это ты паг файл и не мог подключить
